airflow/providers/google/cloud/transfers/sftp_to_gcs.py

```diff
@@ -70,6 +70,27 @@ class SFTPToGCSOperator(BaseOperator):
         Service Account Token Creator IAM role to the directly preceding identity, with first
         account from the list granting this role to the originating account (templated).
     :param sftp_prefetch: Whether to enable SFTP prefetch, the default is True.
+    :param use_stream: Determines the method of file transfer between SFTP and GCS.
+    - If set to False (default), the file is downloaded to the worker's local storage and 
+      then uploaded to GCS. This may require significant disk space on the worker for large files.
+    - If set to True, the file is streamed directly from SFTP to GCS, which does not consume 
+      local disk space on the worker. 
```

Comment:
```suggestion
        - If set to False (default), the file is downloaded to the worker's local storage and 
          then uploaded to GCS. This may require significant disk space on the worker for large files.
        - If set to True, the file is streamed directly from SFTP to GCS, which does not consume 
          local disk space on the worker. 
```

Should we add an additional indentation here?